script_vts.py

```diff
@@ -203,21 +150,90 @@ def handle_file():
     print(f"[+] The file report can be found as {filename}.txt in {Path.cwd()}")
     return 0
 
+##########################################################
+# HELPERS
+##########################################################
 
-def handle_file_report(report_data_dic, string_argument):
-    """Function used to handle our file report"""
-    response = search_api_request(api_key, string_argument)
-    if response is None:  # If we receive nothing quit with error
-        print("[-] We didn't receive something from our search request.")
-        return 1
-    error, report_data_dic, string_parse_json, res_dic, analysis_stat, res_parse_json = parse_json(report_data_dic, response, string_argument)
-    if error is not True:
-        data_process_variable,  report_data_dic = data_processing(string_parse_json, res_dic, analysis_stat, res_parse_json, report_data_dic)
-        if data_process_variable is True:  # check if we need to call for file reputation
-            response_file_reputation = file_reputation_api_request(api_key, string_argument)
-            report_data_dic.append("[+] Response from File reputation request: \n")
-            report_data_dic.append(response_file_reputation)
-    return report_data_dic
+
+def exception_handler(print_exception=False, exception=""):
+    """This function enhances default Python Error handling
+    It will print the line in code that the error occurred on"""
+    if print_exception is True:
+        print(exception)
+        exc_type, exc_obj, exc_tb = sys.exc_info()
+        del exc_type, exc_obj
+        fname = os.path.split(exc_tb.tb_frame.f_code.co_filename)[1]
+        print("Exception on line: ", exc_tb.tb_lineno, " in ", fname)
+    return None
+
+
+def verify_at_least_one_command_line_argument(arguments):
+    """Function checking the number of arguments"""
+    if len(arguments) > 1:
+        return arguments
+    return None
+
+
+def formating_file():
```

Comment:
What is this function supposed to do??? The function name appears inaccurate.
It looks like it's opening a file and stripping all new lines. Name should match that.

Reply:
Okay no problem ! You already check this function before but the name can be changed for sure!